SpatialGDK/Documentation/content/cloud-dev-workflow.md

```diff
@@ -0,0 +1,47 @@
+# Cloud Development Workflow
+
+<!-- This is a live embed of a google drawing -->
+
+<img src="https://docs.google.com/drawings/d/e/2PACX-1vQRmK7TxLji8pT7erPl54hqMMMDsdosZY1OZ2wuPYLQ23dWIrx86qCHggEeq-XasTCsqRe40fCKQvKN/pub?w=758&amp;h=1162">
+
+You may find the below snippets useful as reference:
+
+### Build server-worker assembly
+
+```
+Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat <YourProject>Server Linux Development <YourProject>.uproject
+```
+
+Replacing **`<YourProject>`** with the name of your Unreal project.
```

Comment:
```suggestion
Replacing `<YourProject>` with the name of your Unreal project.
```
No need for bold when we reference file names, code, command line arguments. Bold for UI references.